sous-chefs/packagecloud
# Migrate cookbook to Sous-Chefs

- Migrate cookbook to Sous-Chefs
- Update README
- Rename the test cookbook to test
- Update tested platforms
  - see kitchen.yml for supported platforms
- Turn on unified_mode for Chef 18 support
- Require Chef 15.3 for unified_mode

Signed-off-by: Dan Webb <dan.webb@damacus.io>
